implementations.md

```diff
@@ -2,6 +2,17 @@
 
 This page contains an alphabetical list of projects and vendors that currently implement Trace Context.
 
+## Azure Monitor
+**Website:** [Azure Monitor](https://azure.microsoft.com/en-us/services/monitor/)
+ 
+**Implementations:**
+.NET,
```

Comment:
would you mind adding links to keep it consistent with the rest of the document?

Reply:
Yes. Waiting for confirmation on specification links before applying.

Reply:
Updated with available links

Reply:
Do you want to wait for more links or PR is ready to be merged?

(we will need more approvals anyway, just asking to understand the status)

Reply:
Yes. I am in discussion with devs and we will probably move to "All technologies supported by Azure Monitor..." and link back to docs.microsoft.com since that is our source of truth for documentation. I should be able to close out by tomorrow.

Reply:
We decided that it is best to navigate the user to docs.microsoft.com